justfile

```diff
@@ -136,6 +136,8 @@ profile +args:
     pyinstrument {{ args }}
 
 docs-apigen *args:
+    cd docs && quarto add machow/quartodoc --no-prompt
```

Comment:
I suppose we can remove this line now since the extension is checked into the repo, but I can do that in a follow-up.